lib/src/node-sass/render.test.ts

```diff
@@ -0,0 +1,722 @@
+// Copyright 2021 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import * as p from 'path';
+import {promises as fs} from 'fs';
+import {RawSourceMap} from 'source-map';
+
+import * as sandbox from '../../../spec/helpers/sandbox';
+import {expectEqualIgnoringWhitespace} from '../../../spec/helpers/utils';
+import {compileString} from '../compile';
+import {render, RenderError, RenderResult} from './render';
+
+describe('render', () => {
```

Comment:
Not something we need to worry about right now, but I dream of a future where the JS API tests are shared between implementations.

Reply:
Yep, I think that would be ideal! Are you envisioning an addition to `sass-spec` that handles testing all of the JS implementation?